README.md

```diff
@@ -576,6 +576,36 @@ database {
 
 By default batch inserts will be processed in blocks of 2000. To modify this value add the field `insert-batch-size` to the `database` stanza.
 
+Cromwell metadata (paths to job results, start and end times, etc.) grow at a significantly faster rate than operations
+data (running workflows, previous call cache results, etc.). To use a separate database for operations and/or metadata,
+under the `database` config section, configure a sub-path for `operations` and `metadata` with custom settings.
+
+```hocon
+database {
+  metadata {
+    profile = "slick.jdbc.HsqldbProfile$"
+    db {
+      driver = "org.hsqldb.jdbcDriver"
+      # Store metadata in a file on disk that can grow much larger than RAM limits.
+      url = "jdbc:hsqldb:file:metadata-db-file-path;shutdown=false;hsqldb.tx=mvcc"
+      connectionTimeout = 3000
+    }
+  }
+  operations {
+    profile = "slick.jdbc.HsqldbProfile$"
+    db {
+      driver = "org.hsqldb.jdbcDriver"
+      # Continue to store operations data in memory.
+      url = "jdbc:hsqldb:mem:operations-schema;shutdown=false;hsqldb.tx=mvcc"
```

Comment:
Cool. I didn't realize you were birdstoning here. Since this comes up sometimes in our "eh, you could *try* that" performance suggestions, at least now it's already reayd to go :)